lib/kafka/broker_pool.rb

```diff
@@ -2,12 +2,14 @@
 
 module Kafka
   class BrokerPool
-    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:)
+    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:, ssl: nil, ssl_context: nil)
```

Comment:
Okay. I'm not familiar with the different use cases – it would be nice if SSL setup was documented in the README with a few use cases. Would also make it easier to review.